src/fights.ts

```diff
@@ -2174,10 +2180,83 @@ export function printEmbezzlerLog(): void {
     HIGHLIGHT
   );
 }
+type FreeKill = { source?: Item; skill: Skill; used: () => boolean };
+const freeKills: FreeKill[] = [
```

Comment:
this is funny to me but i think we actually need to add batoomerang here. this is funny to me because neither of us actually have a batoomerang so i'm not even sure we can test it but if we're including jokester's gun we probably gotta include that as a free fight source for, i don't know, James Spader, returning to kol after a brief hiatus spent filming The Blacklist with their only free fight source being their batoomerang.